src/main/java/com/thebois/views/infoviews/ColonyInventoryView.java

```diff
@@ -0,0 +1,63 @@
+package com.thebois.views.infoviews;
+
+import java.util.Locale;
+import java.util.Map;
+
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.VerticalGroup;
+
+import com.thebois.models.inventory.items.ItemType;
+import com.thebois.utils.StringUtils;
+
+/**
+ * Information view for the colony inventory.
+ */
+public class ColonyInventoryView implements IActorView {
```

Comment:
design

Is there any reason this can't be genericized to just be an `InventoryView` that can render any inventory it is given?

Reply:
I guess 👍